Documentation/admin_guide.md

```diff
@@ -209,3 +209,11 @@ If the request times out, it indicates two possibilities:
 2. the majority of the cluster is not functioning.
 
 If timeout happens several times continuously, administrators should check status of cluster and resolve it as soon as possible.
+
+### Deployment
+
+#### CPU cores
+
+In default, etcd process uses only one CPU core.
```

Comment:
etcd only uses one CPU core by default (go 1.5 might change this default behavior link)